v3/how-to-guides/index.mdx

```diff
@@ -1,120 +1,60 @@
-    to easily get started with this template.
-    "
+<RelatedMaterialBlock
+  text={`All guides about benchmarking and weight configurations for runtime engineers.`}
+  linkText={`Weights`}
+  link={`/how-to-guides/weights`}
+  
 />
-
-## Content structure
-
-The structure of Substrate's how-to guides aims to group guides into categories using a keyword tagging system in the
-front-matter of each guide. For example:
-
-**Simple crowdfund.**
-keywords: _runtime, intermediate, pallet design_
-
-The current groupings are to help organize the repository of how-to guide content. They reflect the different
-areas of development within Substrate:
-
-- **Basics**. Where the really simple guides live, those that can be referenced by more complex ones.
-- **Pallet design**. Everything to do with building custom pallets with or without FRAME.
-- **Weights**. Any content that covers configuring weights for specific use cases.
-- **Testing**. A collection of guides for testing.
-- **Storage migrations**. Anything to do with storage migrations.
-- **Consensus**. Client stuff, bridging, node configurations.
-- **Parachains.** _WIP_
-
-### Keywords
-
-_'basics', 'beginner', 'intermediate', 'advanced', 'runtime', 'pallet design', 'weights', 'fees', 'currency', 'testing', 'storage migration', 'node', 'client', 'consensus', 'proof-of-work'_
-
-## FAQ
-
-**What is the difference between a "how-to guide" and a tutorial?**
-
-A **how-to guide** is an in-depth guide intended for someone who is assumed to have prior Substrate knowledge on how to achieve a specific goal. Information inside a guide is only pertinent to achieving that goal. Anything else is irrelevant.
-
-A **tutorial** is a lesson to teach someone how to achieve something assuming they have no prior knowledge on the subject. They contain more details on the subject; cover breadth (vs. how-to guides which cover depth); and can repeat information across other tutorials.
+<RelatedMaterialBlock
+  text={`Guides to cover different use cases for testing pallets and other runtime logic.`}
+  linkText={`Testing`}
+  link={`/how-to-guides/testing`}
+  
+/>
+<RelatedMaterialBlock
+  text={`A collection of guides to help runtime engineers with different types of storage migrations.`}
+  linkText={`Storage Migrations`}
+  link={`/how-to-guides/storage-migrations`}
+  
+/>
+<RelatedMaterialBlock
+  text={`Discover different ways to implement consensus mechanisms in your runtimes.`}
+  linkText={`Consensus`}
+  link={`/how-to-guides/consensus`}
+/>
+<RelatedMaterialBlock
+  text={`All things related to integrating and extending standalone Substrate chains to parachains.`}
+  linkText={`Parachains`}
+  link={`/how-to-guides/parachains`}
+  
+/>
+<RelatedMaterialBlock
+  text={`Guides for tools that are not included out-of-the-box to help you managing Substrate chains in production.`}
+  linkText={`Tools`}
+  link={`/how-to-guides/tools`}
+/>
```

Comment:
We can iterate on the wording, but ideally we'd have parallel construction for each category (probably starting with a verb).